python/wpt/__init__.py

```diff
@@ -33,6 +33,8 @@ def create_parser():
                         help="Pass preferences to servo")
     parser.add_argument('--layout-2020', '--with-layout-2020', default=False,
                         action="store_true", help="Use expected results for the 2020 layout engine")
+    parser.add_argument('--layout-2013', '--with-layout-2013', default=False,
```

Comment:
I know that this is just here to avoid an error about unknown arguments, but maybe the default should be true just to avoid confusion.

Reply:
If it is default true, then we can get in weird situation with both layout2020 and 2013 being selected (well 2020 would be selected). Also that's how it is done here: https://github.com/servo/servo/blob/master/python/servo/command_base.py#L849.